pandas/_testing/asserters.py

```diff
@@ -609,6 +609,43 @@ def assert_interval_array_equal(
     assert_attr_equal("inclusive", left, right, obj=obj)
 
 
+def assert_interval_array_almost_equal(
+    left, right, rtol, atol, obj="IntervalArray"
```

Comment:
Obj is not needed as long as this is only hit by Intervall dtypes

Reply:
I copied most of the code from assert_interval_array_equal(). I think the obj argument is used in line 642: assert_attr_equal("inclusive", left, right, obj=obj). Should I take out the obj parameter and line 642 altogether?

Reply:
You are not passing anything into this function that is not equal to "IntervalArray", hence we don't need this